content/examples/shortcodes/youtube/en.md

```diff
@@ -21,3 +21,31 @@ If no title is provided, a default of “YouTube Video” will be used.
 ```
 
 {{< youtube id="w7Ft2ymGmfc" title="A New Hugo Site in Under Two Minutes" >}}
+
+```
+{{</* youtube id="w7Ft2ymGmfc" title="A New Hugo Site in Under Two Minutes" image="cow.jpg" */>}}
+```
+
+{{< youtube id="w7Ft2ymGmfc" title="A New Hugo Site in Under Two Minutes" image="cow.jpg" >}}
+
+
+```
+{{</* youtube id="w7Ft2ymGmfc" */>}}
+```
+
+{{< youtube id="w7Ft2ymGmfc" >}}
+
+
+## Attributes
+
+id
+: required - Set the ID of the youtube video you want to show
+
+title
+: optional - Set the title of the video
+
+image
+: optional - Set the preview image of the video
```

Comment:
shouldn't we make this required as we can't lean on the youtube fallback image anymore because they set marketing cookies?